usl_pipeline/usl_lib/usl_lib/readers/polygon_readers.py

```diff
@@ -0,0 +1,67 @@
+import typing
+
+import fiona
+import fiona.crs
+import pyproj
+from shapely.geometry import Polygon
+
+
+SHAPE_FILE_POLYGON_FEATURE_TYPE = "Polygon"
+
+
+def read_polygons_from_shape_file(
+    file_path: str,
+    target_crs: typing.Optional[str] = None,
+    mask_value_feature_property: typing.Optional[str] = None,
+    skip_zero_mask_values: bool = True,
+) -> list[typing.Tuple[Polygon, int]]:
```

Comment:
Things from typing are one of the few exceptions to the 'only import namespaces' rule:
https://google.github.io/styleguide/pyguide.html#2241-exemptions

Reply:
Done